internal/c/parts/video/image/src.c

```diff
@@ -1,34 +1,211 @@
-extern uint32 matchcol(int32 r, int32 g, int32 b);
+//----------------------------------------------------------------------------------------------------
+//    ___  ___   __ _ _  ___ ___   ___                       _    _ _
+//   / _ \| _ ) / /| | || _ \ __| |_ _|_ __  __ _ __ _ ___  | |  (_) |__ _ _ __ _ _ _ _  _
+//  | (_) | _ \/ _ \_  _|  _/ _|   | || '  \/ _` / _` / -_) | |__| | '_ \ '_/ _` | '_| || |
+//   \__\_\___/\___/ |_||_| |___| |___|_|_|_\__,_\__, \___| |____|_|_.__/_| \__,_|_|  \_, |
+//                                               |___/                                |__/
+//
+//  QB64-PE Image Library
+//  Powered by stb_image (https://github.com/nothings/stb) & dr_pcx (https://github.com/mackron/dr_pcx)
+//
+//  Copyright (c) 2022 Samuel Gomes
+//  https://github.com/a740g
+//
+//-----------------------------------------------------------------------------------------------------
 
 #ifndef DEPENDENCY_IMAGE_CODEC
 // Stub(s):
-int32 func__loadimage(qbs *f, int32 bpp, int32 passed);
+int32_t func__loadimage(qbs *f, int32_t bpp, int32_t passed);

```

Comment:
I just noticed, this isn't actually a stub xD It's just a function declaration. Presumably it doesn't matter because nothing should be calling `func__loadimage` if it's disabled, but kinda funny.

That probably makes this a good candidate for compiling separate from `libqb.cpp`, I'll make a separate issue to do that.